pkg/types/installconfig.go

```diff
@@ -79,6 +79,9 @@ type AWSPlatform struct {
 	// Region specifies the AWS region where the cluster will be created.
 	Region string `json:"region"`
 
+	// Tags specifies additional tags for AWS resources created by the cluster.
+	Tags map[string]string `json:"tags,omitempty"`
```

Comment:
nit: Although we mentions `Tags` specifies additional tags for AWS resources. The name implies, this is the complete set of tags that will exist on taggable resource. `ExtraTags` makes it clear that this are in addition to many that we create.

/cc @crawford @yifan-gu

Reply:
> The name implies, this is the complete set of tags that will exist on taggable resource.

Based on offline discussion with @abhinavdahiya and @crawford, I've updated this to `UserTags` in d2da4e7 -> 29fc1b1.  I've tried to put a finger on the trade-offs in the commit message, so let me know if you think that needs tweaks ;).